assets/src/story-embed-block/block/edit.js

```diff
@@ -170,7 +170,7 @@ function StoryEmbedEdit({
   const onResizeStart = () => toggleSelection(false);
   const onResizeStop = () => toggleSelection(true);
 
-  const label = __('Web Story URL', 'web-stories');
+  const label = __('Story URL', 'web-stories');
```

Comment:
?

Reply:
The new block will have an option to enter the story URL, once the user selects the block type 'Story URL'. To make it consistent between the block types, it was discussed to change `Web Story URL` to `Story URL` across all the blocks.